maintainers/maintainer-list.nix

```diff
@@ -520,6 +520,12 @@
     githubId = 293191;
     name = "Andres Loeh";
   };
+  andresilva = {
+    email = "andre.beat@gmail.com";
+    github = "andresilva";
+    githubId = 123550;
+    name = "André Silva";
+  };
```

Comment:
Please make this into a separate commit:

```
maintainers: add <name>
```

Reply:
Done.